src/PhpBrew/VariantBuilder.php

```diff
@@ -1180,6 +1175,10 @@ public function build(Build $build)
         // reset options
         $this->options = array();
 
+        // before return all options, we need to put the PKG_CONFIG_PATH
+        // from variants to shell environment.
+        $build->putPkgConfigPathsEnv();
```

Comment:
The `VariantBuilder::build()` method is meant to build the variant and not produce any side effects. Is it possible to move this method call outside of this method somewhere in the `InstallCommand` closer to where the `ConfigureTask` is executed?

Reply:
Okay, I see.